modules/react/layout/lib/utils/grid.ts

```diff
@@ -45,6 +48,29 @@ export type GridStyleProps = {
   grid?: Property.Grid;
 };
 
+export type GridSpaceProps = {
+  /** sets `gap` property */
+  gridGap?: Property.GridGap | GridSpacePropsValues;
+};
+
+const gridGap = (value: GridSpacePropsValues) => {
+  return {gridGap: spaceTokens[value as keyof CanvasSpace] || value};
+};
+const gridRowGap = (value: GridSpacePropsValues) => {
+  return {gridRowGap: spaceTokens[value as keyof CanvasSpace] || value};
+};
+const gridColumnGap = (value: GridSpacePropsValues) => {
+  return {gridColumnGap: spaceTokens[value as keyof CanvasSpace] || value};
+};
+
+const GridSpaceStyleProps = {
```

Comment:
```suggestion
const gridSpaceStyleProps = {
```
I thought this was a type further down because of the PascalCase.

Reply:
Following up: This still needs resolved.

Reply:
Thought this was added in last commit. Has been updated.